R/count_teleconnections.R

```diff
@@ -136,8 +137,9 @@ count_watershed_teleconnections <- function(data_dir,
         # TELECONNECTION - NUMBER OF CROP TYPES BASED ON GCAM CLASSES. NUMBER OF LAND COVERS.
 
         # get raster values of crops within the watershed.
-        get_raster_val_classes(cropcover_USA, watersheds_city) -> cropcover_ids
+        get_raster_val_classes(cropcover_USA, watersheds_city) -> cropcover_raster
```

Comment:
Please rename this function as it no longer gets raster value classes (e.g., `get_raster_value_matrix`). Update the function documentation accordingly (i.e., the name and details in io.R).